katacomb/katacomb/util/image_util.py

```diff
@@ -4,23 +4,23 @@
 from matplotlib import use
 use('Agg', warn=False)  # noqa: E402
 import matplotlib.axes
-from matplotlib import pylab as plt
+import matplotlib.pyplot as plt
 from mpl_toolkits.axes_grid1 import make_axes_locatable
+import numpy as np
 
 import katsdpsigproc.zscale as zscale
 
 
 DEFAULT_DPI = 96
 
 
-def _prepare_axes(wcs, width, height, image_width, image_height, dpi, slices, trim):
+def _prepare_axes(wcs, width, height, image_width, image_height, dpi, slices, bbox):
     fig = plt.Figure(figsize=(width / dpi, height / dpi), dpi=dpi)
     ax = fig.add_subplot(projection=wcs, slices=slices)
     ax.set_xlabel('Right Ascension')
     ax.set_ylabel('Declination')
-    x_trim, y_trim = image_width * trim, image_height * trim
-    ax.set_xlim(-0.5 + x_trim, image_width - x_trim - 0.5)
-    ax.set_ylim(-0.5 + y_trim, image_height - y_trim - 0.5)
+    ax.set_xlim(-0.5 + bbox[0], bbox[1] - 0.5)
+    ax.set_ylim(-0.5 + bbox[2], bbox[3] - 0.5)
```

Comment:
```suggestion
    ax.set_xlim(-0.5 + bbox[0], bbox[1] + 0.5)
    ax.set_ylim(-0.5 + bbox[2], bbox[3] + 0.5)
```
Because the bbox calculation gives a closed range e.g. if all pixels are finite then bbox[1] will be `image_width - 1` rather than `image_width`.